sdks/nodejs/spec/agonesSDK.spec.js

```diff
@@ -187,6 +187,49 @@ describe('AgonesSDK', () => {
 			}
 		});
 
+		it('calls the server and handles stream write error if callback provided', async () => {
+			let stream = jasmine.createSpyObj('stream', ['write', 'on']);
+			stream.write.and.callFake((chunk, encoding, callback) => {
+				callback('error');
+			});
+			spyOn(agonesSDK.client, 'health').and.callFake(() => {
+				return stream;
+			});
+			try {
+				agonesSDK.health(() => {});
+			} catch (error) {
+				fail();
+			}
+		});
+		it('calls the server and re throws stream write error if no callback', async () => {

```

Comment:
nit: For consistency with other tests, perhaps we should add a newline here to separate tests

Reply:
done